### DISCUSSION TITLE
How to get Search Result Page view like SharePoint out of the box Search

### DISCUSSION THREAD

**Discussion by vishalgoyal2472** [microsoft-search/pnp-modern-search#3821] [General]
Hi
1) can we get our PNP search result page have different verticals like the default search page in SharePoint as below?

2) there is default all sources dropdown n default SharePoint online search , can we get that in our pnp search also?

![image](https://github.com/microsoft-search/pnp-modern-search/assets/36663570/b806cf47-e4f3-4cca-b581-db019abfe219)

**Comment by kasperbolarsen** [comment]
Hi, the PnP Modern Search web parts does contain both a Search Vertical and a Search Refiner web part.

Both can be configured and extended way beyond what the out of the box Search provide 🙂
